debian/libopx-base-model1.install

```diff
@@ -1,3 +1,2 @@
 usr/lib/*/*.so.*
 usr/lib/*/*.so
-usr/share/opx/yang-models
```

Comment:
Do not remove this.  The OPX package must include the YANG models -- they are needed by OS10.